dev-support/docker-images/jupyter-gpu/Dockerfile

```diff
@@ -117,7 +118,7 @@ RUN git clone --depth=1 https://github.com/apache/submarine && \
     rm submarine -rf
 
 # Add qlib example in notebook
-RUN wget https://raw.githubusercontent.com/microsoft/qlib/main/examples/workflow_by_code.ipynb -P $HOME
+RUN wget https://raw.githubusercontent.com/microsoft/qlib/v0.9.0/examples/workflow_by_code.ipynb -P $HOME
```

Comment:
Is it possible to keep the version of qlib the same as the current file? I've tried and I think version 0.8.6 is also available for download.
https://raw.githubusercontent.com/microsoft/qlib/v0.8.6/examples/workflow_by_code.ipynb

Others LGTM